### DISCUSSION TITLE
Rename MacOS Helpers

### DISCUSSION THREAD

**Discussion by ayushmanchhabra** [nwutils/nw-builder#930] [Q&A]
Refs: https://github.com/nwutils/nw-builder/issues/897#issuecomment-1630190132
Refs: #705

@zkrige I don't have a MacOS to test this on. Are you up for creating a PR to test this?

**Comment by sysrage** [comment]
Changing the copyright string is much different than renaming the helpers. I successfully tested renaming the copyright string. When I tried renaming the helpers, it fails. You can even find code that used to do this in the nw-builder commit history. 😁

**Reply by ayushmanchhabra** [reply]
Did you rename the Helpers and update the Plist values which reference those Helpers?

**Reply by sysrage** [reply]
No, I don't think I updated Plist values. I believe we were only renaming the files and trying `product_string`. If you have code that you think should work, I'm happy to give it a try!

This was part of the conversation: https://github.com/nwutils/nw-builder/issues/705#issuecomment-1328170356

The related NW.js issue: https://github.com/nwjs/nw.js/issues/7253

If you look at the nw-builder code from around the time of the above issue/comment, we had a bunch of this implemented but I believe you later removed it.

**Reply by ayushmanchhabra** [reply]
Awesome, I'll come up with a PR soon
